### ISSUE TITLE
How to install legacy HHVM 3.x?

### ISSUE THREAD

**Issue by clue** [azjezz/setup-hhvm#1]
Thanks for providing this excellent action, really helps setting up specific HHVM versions!

I've got this to successfully install newer HHVM 4.x versions, but have yet to find a way to install legacy HHVM 3.x.

As per https://docs.hhvm.com/hhvm/installation/linux#obtaining-a-specific-release the version should be `3.30-lts` instead of `3.30`. This does however yield an error because `apt` can not fetch the distribution. https://dl.hhvm.com/ubuntu/dists/ suggests this should probably be `lts-3.30` instead, so I suppose this is an error in HHVM's documentation. With the `lts-3.30` version specified, `apt` can indeed fetch the repository, but refuses to install because a newer version is already installed:

```
…
Reading package lists...
+ '[' false = true ']'
+ sudo apt-get install -y hhvm
Reading package lists...
Building dependency tree...
Reading state information...
hhvm is already the newest version (4.84.0-1~bionic).
0 upgraded, 0 newly installed, 0 to remove and 36 not upgraded.
```

I have yet to find a good way to allow this action to install legacy HHVM.

In the meantime, I'm manually downloading the required `deb` file and manually install it through `apt` as per: https://github.com/clue/reactphp-flux/pull/21/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR39-R40

**Comment by azjezz** [comment]
I think the issue is this action doesn't use `--allow-downgrades`, as GitHub action containers have HHVM pre-installed on them (  https://github.com/actions/virtual-environments/blob/f38833acec5cc1c3239474f8135b23c17cb72351/images/linux/scripts/installers/hhvm.sh -  hence "hhvm is already the newest version" message )

I will try to add `--allow-downgrades` option and see if this solves your problem 👍

**Comment by azjezz** [comment]
fixed in #2 👍  thank you for reporting 🎉 

( the issue was as expected from GitHub pre-installed hhvm binary, removing it allows installing older hhvm builds. )
